### DISCUSSION TITLE
Materialize3-theme for stash

### DISCUSSION THREAD

**Discussion by killhellokitty** [stashapp/stash#4438] [Show and tell]
I have been working on this theme for little over a year now.  
Is there anyone interested in a "material3" design theme for Stash? let me know.  
Previously I worked on several GTK themes my GitHub is killhellokitty

The theme is actually a near complete rewrite of the CSS for Stash.

**Comment by stg-annon** [comment]
stash allows for CSS Themes or overrides, you can see example themes [here](https://github.com/stashapp/CommunityScripts/tree/main/themes) if you want to contribute to UX design of stash's main branch there is a discord channel for it but discussions here work as well, you might just get less feedback

**Comment by cj12312021** [comment]
As `stg-annon` mentioned, we certainly welcome new themes. I know some users on the discord have specifically requested a material design styled theme.

**Reply by killhellokitty** [reply]
Could you please direct me to the Discord page?

**Reply by HereIam80** [reply]
https://discord.gg/DtvCvaAw

**Reply by DogmaDragon** [reply]
https://discord.gg/frQ7qBvB3S

**Comment by HereIam80** [comment]
Where do I apply to check it out?

**Reply by killhellokitty** [reply]
(https://github.com/killhellokitty/stash-material-ize-theme)

**Comment by killhellokitty** [comment]
the first near complete draft is now published on my github if anyone is interested in trying it out and giving some feed back, it would be greatly appreciated.
my github is killhellokitty/stash-material-ize-theme (https://github.com/killhellokitty/stash-material-ize-theme)

**Reply by Chaprnks** [reply]
why did you delete the repository??

**Comment by killhellokitty** [comment]
It is recommended you use Chrome or Firefox.
There is still a lot of cleanup work being done in the edit areas, please be patient :-)

**Comment by cj12312021** [comment]
@killhellokitty I really like the direction the theme is going. You added nice touches in certain areas I can't help but appreciate, like on the scene player page with the scene divider. I also recently made changes to that page to make the details section sit on the page more naturally. It is clear that some areas are still in development, so I'll be interested in checking back to see what the finished product would look like. So far, though, this is among one of the best themes I've seen. 

I don't really like critiquing themes since the design will usually be based on the user's preference, but assuming most of the UI quirks/issues I saw are already on your radar, my only comment would be around reducing the font size on the top nav bar and the size of the font on the settings pages. The font size, as is, eats up too much screen real estate, especially considering the side nav you have (which I do like).

When you're done, I'd encourage you to put a pull request out to the https://github.com/stashapp/CommunityScripts repo and submit your theme to the `theme` folder if you want to make the theme more accessible to users.

**Comment by feederbox826** [comment]
repo deleted
